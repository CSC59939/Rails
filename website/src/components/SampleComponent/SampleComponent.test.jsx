
import React from 'react';
import renderer from 'react-test-renderer';
import { SampleComponent } from '..';

test('Expect SampleComponent to return a div with its name', () => { // eslint-disable-line no-undef
  const component = renderer.create(
    <SampleComponent />,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot(); // eslint-disable-line no-undef
});
