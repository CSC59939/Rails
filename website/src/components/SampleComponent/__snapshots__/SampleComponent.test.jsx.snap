// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect SampleComponent to return a div with its name 1`] = `
<div>
  A sample Component
</div>
`;
